### ISSUE TITLE
is the project alive? // questions about python and package utility

### ISSUE THREAD

**Issue by ink-splatters** [ColdGrub1384/LibTerm#85]
Hello, thanks for the cool project.

I’m wondering if the project is alive… as it seems too many issues don’t get commented…

I have a couple of questions:

-any plans to update python from 3.7 to reasonable nowadays version?

-where is pip? o_O how is the whole thing  supposed to be used? Unfortunately package util is hardly an option and it doesn’t allow extensions except for DNS hijacking (for pointing to custom repo).

**Comment by noahbroyles** [comment]
Hey there,

I am not associated with this repository at all, so I don't know if it's alive or not. But I know that you can install Python3 packages with pip like this:
```console
python3 -m pip install <package>
```

**Comment by ink-splatters** [comment]
Indeed: very standard way which I’d less likely use in practice… falsely assuming symlinks would have been also there.

Still, running binaries seems obviously not to be an option, so a lot of packages falls short; and it’s very understandable. Anyway, there are plenty of pure Python gems to be happy about.

Probably, with a fair bit of hacking around iOS JIT (which is still, AFAIK, possible to use without jailbreak, given extra investments into Apple : by either having 2 devices or paid Dev Account: https://github.com/osy/Jitterbug is build around those options).

If successful, it might become possible also to just-in-time recompile arm64 binaries, also native arm64 Mach exes which are parts of Python packages.

Anyone interested in the topic and / or passers-by experienced with  iOS JIT (which, BTW, I’m not) are very welcome to join the discussion (or be the cause for it to stop abruptly if it seems not to be viable idea).


Br,
is
